packages/scxa-tsne-plot/__test__/ClusterTSnePlot.test.js

```diff
@@ -94,16 +94,18 @@ describe(`ClusterTSnePlot`, () => {
     },
     plotTypeDropdown: [
       {
-        plotType: `TSne`,
+        plotType: `tSNE`,
```

Comment:
A better approach in my view, load the backend response dynamically into `Plotype` and `Options` dropdowns without changing much. So that in the future, if we added more projection methods and its options, no need to change the front-end code.